src/endpoints/http/collections/index.js

```diff
@@ -1,9 +1,10 @@
 import listCollections from './list';
 import {createCollection} from './create';
+import authorize from '../../../middleware/route-authorize';
 
 export function collectionsHandler(router) {
   //list collection info
-  router.get('/collections', function(req, res, next) {
+  router.get('/collections', authorize({access: 'read'}), function(req, res, next) {
```

Comment:
Yes the terminology here is important.
I was separating the permission into the resource e.g. `cluster/reseller/customer/domain/service/data-browser` and the access e.g. `read`, `write` etc. The caller would need the correct permission + access. In this instance we are defining the permission globally for the service in the config file so that's why passing access here seemed correct to me. So we are saying allow read access on this resource/route.

Any suggestions are welcome as I was unsure of this terminology.